SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -205,15 +207,38 @@ public class SwiftyTesseract {
     }
     
     // create unique file path
-    let filepath = try processPDF(images: images)
-    
+    let filepath = URL(fileURLWithPath: NSTemporaryDirectory()).appendingPathComponent(UUID().uuidString)
+    try processPDF(images: images, at: filepath)
```

Comment:
We actually should change this assignment back to `let filepath = try processPDF(images: images)`